tests/pytorch/test_pytorch_model_export.py

```diff
@@ -769,6 +790,41 @@ def test_load_model_succeeds_when_data_is_model_file_instead_of_directory(
     )
 
 
+def compare_state_dicts(state_dict1, state_dict2):
+    for key, _ in state_dict1.items():
+        if key not in state_dict2:
+            return False
+        value1 = state_dict1[key]
+        value2 = state_dict2[key]
+        if isinstance(value1, torch.Tensor) and isinstance(value2, torch.Tensor):
+            if not torch.all(value1.eq(value2)):
```

Comment:
Can we use `torch.equal` (which I think is easier to understand) here?

https://pytorch.org/docs/stable/generated/torch.equal.html